.editorconfig

```diff
@@ -18,6 +18,10 @@ indent_style = space
 indent_size = 2
 tab_width = 2
 
+# S3872 // Interfaces should not be empty
```

Comment:
This doesn't appear to be the right description for what the original change was?

It's better (IMHO) to suppress these in the code with `#pragma warning disable/restore` pairs at the specific points of usage, rather the hiding it away in the editorconfig.

Reply:
You are right, in this case we don't have many violations for this rule, so it makes sense to keep it in code. Also, it can serve as a reminder to fix it in the future.